packages/utils/src/fhir/fhir-converter/integration-test.ts

```diff
@@ -121,19 +121,26 @@ export async function main() {
   });
   console.log(`Found ${ccdaFileNames.length} XML files.`);
 
-  let nonConvertibleDocumentCount = 0;
+  const nonConvertibleDocMap = new Map<string, string[]>();
   const clinicalDocuments = ccdaFileNames.flatMap(f => {
     const fileContents = getFileContents(f);
-    const isConvertibleResult = isConvertible(fileContents, f);
+    const isConvertibleResult = isConvertible(fileContents);
     if (!isConvertibleResult.isValid) {
-      nonConvertibleDocumentCount++;
+      const existingReasons = nonConvertibleDocMap.get(isConvertibleResult.reason) ?? [];
+      if (!existingReasons.includes(f)) {
+        nonConvertibleDocMap.set(isConvertibleResult.reason, [...existingReasons, f]);
+      } else {
+        nonConvertibleDocMap.set(isConvertibleResult.reason, [f]);
+      }
```

Comment:
Defs NABD (it works and this is utils), but for code approach and simplicity, it would be much simpler to store the count of non-convertible files, since we don't use the list of files.